pkg/controller/clusters/status.go

```diff
@@ -210,3 +226,12 @@ func updateWorkersReadyCondition(cluster *anywherev1.Cluster, machineDeployments
 func controlPlaneInitializationInProgressCondition() *anywherev1.Condition {
 	return conditions.FalseCondition(anywherev1.ControlPlaneInitializedCondition, anywherev1.ControlPlaneInitializationInProgressReason, clusterv1.ConditionSeverityInfo, "The first control plane instance is not available yet")
 }
+
+// SetClusterDefaultCNI sets the DefaultCNI for the Cluster status.
+func SetClusterDefaultCNI(cluster *anywherev1.Cluster, status anywherev1.ClusterCNIStatus, version string) {
```

Comment:
do we need a method for this?

Reply:
it can be removed, and we could set the field directly